ngraph/frontend/paddlepaddle/include/paddlepaddle_frontend/model.hpp

```diff
@@ -14,6 +14,12 @@ namespace ngraph
         class OpPlacePDPD;
         class TensorPlacePDPD;
 
+#if defined(ENABLE_UNICODE_PATH_SUPPORT) && defined(_WIN32)
+        typedef std::wstring path_type;
+#else
+        typedef std::string path_type;
+#endif
+
         class PDPD_API InputModelPDPD : public InputModel
```

Comment:
@slyalin why do we need specific input models like `InputModelPDPD` in public API? should not end users work with any model from any framework in uniform manner?

The same for frontends, places, etc I except

Reply:
Right now I don't see cases when it is necessary. But if we will use it for direct PDPD integration we _may want_ to extend API for specific internal usages. So for now it can be hidden but why do we need to hide it?

Reply:
Because you introduce explicit public API which is not currently needed

Reply:
I think it can be developer API. Why do we need to have it in Public API?

Reply:
Can we make public API more clear and remove classes which won't be used by end user?